Platforms/QemuSbsaPkg/NugetPublishing/DebugConfig.yaml

```diff
@@ -0,0 +1,9 @@
+author_string: Microsoft Core UEFI
+copyright_string: Copyright (c) Microsoft Corporation.
+description_string: QEMU Q35 SBSA DEBUG Platform Firmware
+name: Mu.QemuQ35SBSA.FW.DEBUG

```

Comment:
use of Q35 for SBSA here as well

Reply:
Updated in PR commit https://github.com/microsoft/mu_tiano_platforms/pull/286/commits/3ba367854e17ccff03aa14b948fc471232c5031d